src/LogFileReader.cpp

```diff
@@ -75,11 +84,17 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
-
+    QStringList lines = line.split(COMMA);
+    QTime time = QTime::fromString(lines[0],TIMEFORMAT);
+    if (time.isValid())
+    {
+        batteryData.time = time;
+    }
+    batteryData.voltage = lines[1].toDouble();
+    batteryData.current = lines[2].toDouble();
     // This is here to the compiler happy. Otherwise the compile
```

Comment:
now that we've removed these lines for the unused variables, as above we can clean this up by removing the comments.